contrib/kube-prometheus/assets/prometheus/rules/etcd3.rules

```diff
@@ -0,0 +1,177 @@
+# general cluster availability
```

Comment:
these are the general etcd3 ones, I was thinking of not vendoring in these rules from here as I don't think we can assume a self-hosted etcd everywhere